src/qililab/__init__.py

```diff
@@ -16,14 +16,14 @@
 This is the top level module from which all basic functions and classes of
 Qililab can be directly imported.
 """
+from .circuit_transpiler import CircuitTranspiler, Drag, Wait
```

Comment:
Do we need `CircuitTranspiler` here? Are users supposed to use this class directly? They should use platform instead, right?